website/search/share_search.py

```diff
@@ -220,3 +230,67 @@ def data_for_charts(elastic_results):
     }
 
     return all_data
+
+
+def atom(name, data, query, size, start, url):
```

Comment:
I would take this logic, and put it in the utils file. The functionality that should be left here is converting a single json item to an atom item. Then, you can pass that function into the utils function (so it takes some elasticsearch results, and a function to convert a single result to a feed item). Then in the utils function, you have 

```python
def atom(..., results, to_atom):
...  # set up the feed
    for doc in data:
        feed.add(**to_atom(doc))
...  # returning/etc
```

and in the ```share_search.to_atom``` function you have
```python
def to_atom(result):
    return {
        'title': # title logic,
        'content': # content logic,
        ...
    }
```

That way any other atom feeds we add in the future will have to define very little logic (because pagination, titling, etc can be handled in the generic utils function).